.github/workflows/daily_update.yml

```diff
@@ -22,9 +25,3 @@ jobs:
           git add -A
           git diff --quiet && git diff --staged --quiet || (git commit -m "Daily update from Dark Visitors" && git push)
```

Comment:
The above commit message is not appropriate for a merge of a PR. I wonder if we need to keep the actions separate and use different commit messages in each case?

Reply:
I agree. This is a new inconsistency. I'll see what I can do.

Reply:
Thanks.